honeycomb-beeline.gemspec

```diff
@@ -37,7 +37,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "libhoney", ">= 1.14.2", "~> 1.14"
```

Comment:
you might want a `< 3` in here to prevent using potentially breaking changes in the future?

Reply:
@MikeGoldsmith and I talked about that and see that point. In the end though, we concluded that we'd rather put a guard in later if necessary than require a Beeline upgrade to use a new libhoney. Particularly in cases like this, where the major version bump for libhoney was not a change to the public API, but Ruby version drop-support.